tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -37,6 +38,96 @@ func init() {
 	grpclog.SetLogger(plog)
 }
 
+type stressFunc func(ctx context.Context, kvc pb.KVClient, key, val []byte) error
```

Comment:
This parameterization seems too brittle. Just `type stressFunc func(ctx context.Context) error`? That way the function has full control over all the details of its behavior. For example,
```go
func newStressPut(kvc pb.KVClient, keySuffixRange, keySize int64) stressFunc {
    return func(ctx context.Context) error {
        _, err := kvc.Put(putctx, &pb.PutRequest{
            Key:   []byte(fmt.Sprintf("foo%d", rand.Intn(keySuffixRange))),
            Value: []byte(randStr(keySize)),
        }
        return err
    }
}
```